src/components/smart-container/smart-container-item.js

```diff
@@ -21,7 +21,7 @@ class SmartContainerItem extends Component {
   render(props: any): React$Element<any> {
     return (
       <div className={[style.smartContainerItem, style.selectMenuItem].join(' ')}>
-        <label htmlFor={props.label.toLowerCase()}>
+        <label htmlFor={props.label && props.label.toLowerCase()}>
```

Comment:
I suggest you do the check once before the return and then apply it in the jsx instead of repeating this twice